websites/S/SlimeRead/metadata.json

```
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"name": "Luscarvalho",
+		"id": "458776046184169473"
+	},
+	"service": "SlimeRead",
+	"description": {
+		"en": "Discover and enjoy a vast collection of literary works, including manga, manhwa, manhua, novels, and much more.",
+		"pt_BR": "Descubra e desfrute de uma vasta coleção de obras literárias, incluindo mangás, manwhas, manhuas, novels e muito mais."
+	},
+	"url": [
+		"slimeread.com",
+		"www.slimeread.com"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/Ht1LSYT.png",
```

This is without the background:
```suggestion
	"logo": "https://i.imgur.com/x6NuCdk.png",
```